### PR TITLE
Correct a few typos

### PR DESCRIPTION
**Pull request by duboism**
This PR corrects a few typos that we found while attending the ODW.

If you have suggestions, don't hesitate.

### CODE REVIEW

File: Tutorials/Day_3/Tuto_3.1_Introduction_to_parameter_estimation.ipynb [hunk 1]
```diff
@@ -1128,8 +1128,8 @@
     "\n",
     "However, it is fun to also show how we can quantify this using the posterior odds (defined in the first section!). For this, we need to calculate the evidence $\\mathcal{Z}(d | M_s)$ and $\\mathcal{Z}(d | M_t)$. They can be estimated by stochastic sampling. However, `bilby_mcmc` is not the right tool (in can in principle, but the alternative is superior for evidence calculation). Instead, we should use the **nested sampling** package `dynesty`\n",
     "\n",
-    "- Rerun your analysis of both models, but use `sampler=\"dynesty\"` in `run_sampler`. You will also need to add configuration arguments: `nlive=500, sampler=\"rwalk\"`)\n",
-    "- Calculate the Bayes factor using `np.exp(result_Mt.log_evidence - result_Mt.log_evidence)`"
+    "- Rerun your analysis of both models, but use `sampler=\"dynesty\"` in `run_sampler`. You will also need to add configuration arguments: `nlive=500, sample=\"rwalk\"`)\n",
```
Comment (martinberoiz) [inline Tutorials/Day_3/Tuto_3.1_Introduction_to_parameter_estimation.ipynb:1131]: I don't know dynesty enough, but I think the word `sampler` was correct here. They advice to use the `rwalk` sampler, not sample.

**Review by martinberoiz — CHANGES_REQUESTED** [review]
It's all good except for one comment at the end, about the word `sampler`.

**Comment by jkanner** [comment]
I think Martin is right about `sample` vs `sampler`.  Otherwise, this seems fine.

File: Tutorials/Day_3/Tuto_3.1_Introduction_to_parameter_estimation.ipynb (hunk 1 above)
Comment (martinberoiz) [reply Tutorials/Day_3/Tuto_3.1_Introduction_to_parameter_estimation.ipynb:1131]: Yeap, seems you're right. Then it all looks good to me.

**Review by martinberoiz — APPROVED** [review]

File: Tutorials/Day_3/Tuto_3.1_Introduction_to_parameter_estimation.ipynb (hunk 1 above)
Comment (duboism) [reply Tutorials/Day_3/Tuto_3.1_Introduction_to_parameter_estimation.ipynb:1131]: Thanks for the confirmation.

**Comment by duboism** [comment]
Martin and I agreed on the `sample` vs `sampler` issue so I merged the PR. Thanks for the reviews.

**Pull request merged**